controllers/pullRequestsController.js

```diff
@@ -107,7 +107,8 @@ const getStalePRs = async (req, res) => {
  */
 const getOpenPRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchOpenPRs()
+    const pageNumber = ((req.query.page) || 1)
+    const { data } = await githubService.fetchOpenPRs(pageNumber)
```

Comment:
Can we simply pass it to the function than saving it in variable and then passing it?
`fetchOpenPRs(req.query.page || 1)` would be short and sweet